src/applications/disability-benefits/781/helpers.js

```diff
@@ -35,3 +30,80 @@ export const introductionText = (
     </p>
   </div>
 );
+
+export const ptsdTypeDescription = () => {
+  return (
+    <div>
+      <p>
+        First we're going to ask you about the type of event or events that
+        contributed to your PTSD.
+      </p>
+      <p>
+        What type of event contributed to your PTSD? (Choose all that apply.)
+      </p>
+    </div>
+  );
+};
+
+export const ptsdTypeHelp = () => {
+  return (
+    <AdditionalInfo triggerText="Which should I choose?">
+      <h4>Types of Stressful incidents</h4>
+      <h5>Combat</h5>
+      <p>
+        This means you participated in a fight or encounter with a military enemy
+        or hostile unit or weapon. It also includes if you were present during these
+        events either as a combatant or a Servicemember supporting combatants such
+        as providing medical care to the wounded.
+      </p>
+      <h5>Military Sexual Trauma</h5>
+      <p>
+        This includes sexual harassment, sexual assault, or rape that happens in
+        a military setting.
+      </p>
+      <h5>Personal Assault</h5>
+      <p>
+        This means a person, who isn't part of an enemy force, committed harm.
```

Comment:
Curly apostrophe -> Alt + ] ‘

Reply:
Please never actually use that. If you want a 'safe' apostrophe, please use `&39;`.

See [this](https://www.w3.org/MarkUp/html3/latin1.html) for a list of available HTML Entities.

Reply:
@igreulich : Thanks! She just said to use curly apostrophe. So that'w what I did. Will use what you said.